.buildkite/inference-benchmark-pipeline.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -euf
+
+# This script is executed by Buildkite to generate pipeline YAML string.
+# Document: https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines
+#
+# Usage: CONVERT_RESULT_PATH=gs://path/to/result.tgz ./inference-benchmark-pipeline.sh
+
+# test cases to run inference
+# "<buildkite agent-type> <model filename>"
+TEST_CASES=(
+    "de10nano       lib/lib_arm.so"
+    "de10nano       lib/lib_fpga.so"
+
+    "jetson-nano    lib/lib_aarch64.so"
```

Comment:
aarch64 for Jetson is in the list of benchmark?
Raspberry is in the list.

Reply:
Sorry, this was my mistake. Only GPU is enough for Jetson devices. But aarch64 for Raspberry Pi is needed.

Reply:
I see.